src/views/research/l10n.json

```diff
@@ -1,9 +1,9 @@
 {
     "research.title":"Research on Scratch",
-    "research.conductors":"Research on Scratch is being conducted by members of the MIT Scratch Team and researchers at other universities, including Yasmin Kafai at the University of Pennsylvania Graduate School of Education, Karen Brennan at the Harvard Graduate School of Education, Benjamin Mako Hill at the University of Washington, Andrés Monroy Hernandez at Microsoft Research, Mimi Ito and Crystle Martin at the University of California, Irvine, Quinn Burke at College of Charleston, Deborah Fields at Utah State University, and Kylie Peppler at Indiana University.",
-    "research.privacy":"By sharing projects and participating in the Scratch online community, you are helping us better understand how people can use and learn with Scratch. Any publicly shared projects, comments, or other material on the Scratch site may be included in the research analysis, presentations, papers, and reports. No personally identifiable information is shared. (If you have any questions, please use the {contactLink} form.)",
+    "research.conductors":"Research on Scratch is being conducted by the Scratch Foundation team. By sharing projects and participating in the Scratch online community, you are helping us better understand how people use and learn with Scratch. Any publicly shared projects, comments, or other material on the Scratch site may be included in research analyses, presentations, and reports. We never share personally identifiable information. To learn more, please read the {ethicsCodeLink}. (If you have any questions, please use the {contactLink} form.)",
+    "research.privacy":"Previous research collaborators have included the Lifelong Kindergarten group at MIT, Yasmin Kafai at the University of Pennsylvania Graduate School of Education, Karen Brennan at the Harvard Graduate School of Education, Benjamin Mako Hill at the University of Washington, Andrés Monroy Hernandez at Microsoft Research, Mimi Ito and Crystle Martin at the University of California, Irvine, Quinn Burke at College of Charleston, Deborah Fields at Utah State University, and Kylie Peppler at Indiana University. Below are selected research papers, presentations, and theses on Scratch and the Scratch online community, followed by a list of National Science Foundation grants awarded to support Scratch.",
```

Comment:
Maybe we should change this id name to reflect the new content-- "research.collaborators" maybe?